node/Cargo.toml

```diff
@@ -3,101 +3,104 @@ name = "zeitgeist"
-sp-keystore = { branch = "polkadot-v0.9.23", git = "https://github.com/paritytech/substrate", optional = true }
-substrate-prometheus-endpoint = { branch = "polkadot-v0.9.23", git = "https://github.com/paritytech/substrate", optional = true }
+session-keys-primitives = { tag = "v0.26.1", default-features = false, git = "https://github.com/purestake/moonbeam", optional = true }
+sp-keystore = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate", optional = true }
+substrate-prometheus-endpoint = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate", optional = true }
 
 # Polkadot
 
-polkadot-cli = { branch = "release-v0.9.23", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-parachain = { branch = "release-v0.9.23", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-primitives = { branch = "release-v0.9.23", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-service = { branch = "release-v0.9.23", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-test-service = { branch = "release-v0.9.23", git = "https://github.com/paritytech/polkadot", optional = true }
+polkadot-cli = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
+polkadot-parachain = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
+polkadot-primitives = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
+polkadot-service = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
+polkadot-test-service = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
 
 # Standalone
 
-sc-consensus-aura = { branch = "polkadot-v0.9.23", git = "https://github.com/paritytech/substrate" }
-sc-finality-grandpa = { branch = "polkadot-v0.9.23", git = "https://github.com/paritytech/substrate" }
-sp-consensus-aura = { branch = "polkadot-v0.9.23", git = "https://github.com/paritytech/substrate" }
-sp-finality-grandpa = { branch = "polkadot-v0.9.23", git = "https://github.com/paritytech/substrate" }
+sc-consensus-aura = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
+sc-finality-grandpa = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
+sp-consensus-aura = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
+sp-finality-grandpa = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
 
 # Utility
 cfg-if = { version = "1.0.0" }
 clap = { version = "3.1.18", features = ["derive"] }
 hex-literal = { version = "0.3.4" }
-jsonrpsee = { version = "0.13.0", features = ["server"] }
+jsonrpsee = { version = "0.14.0", features = ["server"] }
 log = { optional = true, version = "0.4.17" }
+# TODO: Remove in future Polkadot release
```

Comment:
Let's create an issue for that. 

And then append it to the TODO.
`# TODO(#issue_number): ...`